src/dpos-user.ts

```diff
@@ -213,7 +199,7 @@ export class DPOSUser {
     this._dappchainGateway = dappchainGateway
     this._dappchainLoom = dappchainLoom
     this._dappchainDPOS = dappchainDPOS
-    this._dappchainMapper = dappchainMapper
+    this._dappchainMapper = dappchainMapper!
```

Comment:
Then `!` shouldn't be required since `null` is an acceptable value for this property?

Reply:
Nope the compile complains on not explicitly saying `!`, ```
Type 'AddressMapper | null' is not assignable to type 'AddressMapper'.
  Type 'null' is not assignable to type 'AddressMapper'.```

Reply:
Ok, now the property `dappchainMapper` is explicitly saying that can be `null` too